PocketKit/Sources/SaveToPocketKit/SavedItem/SavedItemViewModel.swift

```diff
@@ -150,24 +150,8 @@ extension SavedItemViewModel {
         }
 
         for provider in providers {
-            let plainTextUTI = "public.plain-text"
-            let urlUTI = "public.url"
-
-            if provider.hasItemConformingToTypeIdentifier(plainTextUTI) {
-                guard let string = try? await provider.loadItem(forTypeIdentifier: plainTextUTI, options: nil) as? String,
-                      let url = retrieveURLFromString(with: string) else {
```

Comment:
should we remove `retrieveURLFromString` since we're not using that implementation?